src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/GetRefreshUsersResponse.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.professionalapi.controller.response;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.hmcts.reform.professionalapi.domain.RefreshUser;
+
+import java.util.List;
+import java.util.UUID;
+
+@Setter
+@Getter
+@NoArgsConstructor
+public class GetRefreshUsersResponse {
+
+    private List<RefreshUser> users;
+    private UUID lastRecordInPage;
+    private boolean moreAvailable;
+
+    public GetRefreshUsersResponse(List<RefreshUser> users, UUID lastRecordInPage, boolean moreAvailable) {
```

Comment:
We can use  **import lombok.AllArgsConstructor;**

Reply:
added in [a4aa031](https://github.com/hmcts/rd-professional-api/pull/1529/commits/a4aa0314c9fdcfa34d663b7aa5778296934b4e2b)